Gemfile.lock

```diff
@@ -66,7 +64,6 @@ GEM
       rack (~> 1.5)
       rack-protection (~> 1.4)
       tilt (>= 1.3, < 3)
-    thor (0.19.1)
```

Comment:
This is still a dependency:
https://github.com/ddollar/foreman/blob/master/foreman.gemspec#L20